src/com/xilinx/rapidwright/eco/ECOTools.java

```diff
@@ -1062,6 +1106,68 @@ public static void createCell(Design design, EDIFCell reference, List<String> pa
         }
     }
 
+    /**
+     * Creates and places a new primitive cell inline from the existing net driving
```

Comment:
Can you comment where this cell will be instantiated? Is it in the same level of hierarchy as the input pin? And that the cell/nets will have the prefix `inline_insertion_`.